tests/src/core/testqgsvectortilelayer.cpp

```diff
@@ -75,6 +76,24 @@ void TestQgsVectorTileLayer::initTestCase()
   mMapSettings = new QgsMapSettings();
   mMapSettings->setLayers( QList<QgsMapLayer *>() << mLayer );
 
+  // let's have some standard style config for the layer
+  QColor polygonFillColor = Qt::blue;
+  QColor polygonStrokeColor = polygonFillColor;
+  polygonFillColor.setAlpha( 100 );
+  double polygonStrokeWidth = DEFAULT_LINE_WIDTH;
+  QColor lineStrokeColor = Qt::blue;
+  double lineStrokeWidth = DEFAULT_LINE_WIDTH;
```

Comment:
Could you fatten this? Looking at the reference image, it has a lot of very fine linework and is going to be a source of platform related rendering differences...